docs/parameters.yaml

```
@@ -148,6 +148,74 @@ type: filename
 default: none
 components: ["*"]
 ---
+name: Logging.XrootdPssSetOptOrigin
+description: >-
+  Configures the XRootD client used by the proxy top communicate with the origin (we use to set debug level).
+  Values include: DebugLevel (0, 1, 2, 3)
+type: string
+default: DebugLevel 1
+components: ["cache"]
+---
+name: Logging.XrootdPssSetOptCache
+description: >-
+  Configures the XRootD client used by the proxy top communicate with the cache (we use to set debug level).
+  Values include: DebugLevel (0, 1, 2, 3)
+type: string
+default: DebugLevel 3
+components: ["cache"]
+---
+name: Logging.XrootdPssTrace
+description: >-
+  Logging level of pss within Xrootd configuration values include: all, on, debug
+type: string
+default: all
+components: ["cache"]
+---
+name: Logging.XrootdScitokensTrace
+description: >-
+  Trace level of scitokens debug output within Xrootd configuration. Values include: all, on, debug
```

```suggestion
  Trace level of scitokens debug output within Xrootd configuration. Values include: all, debug, info, warning, error, none
```
(from https://github.com/xrootd/xrootd/blob/8f8498d66aa583c54c0875bb1cfe432f4be040f4/src/XrdSciTokens/XrdSciTokensAccess.cc#L951-L963)